terraform/modules/cloudwatch_s3_ingestor/firehose.tf

```diff
@@ -20,7 +20,9 @@ resource "aws_kinesis_firehose_delivery_stream" "cloudwatch_stream" {
   name        = "${var.name_prefix}-${each.key}-delivery-stream"
   destination = "extended_s3"
 
-
+  server_side_encryption {
+      enabled = true
```

Comment:
This is fine, but going forward, I think we might want to work on using customer managed keys which give us a lot more control

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/kinesis_firehose_delivery_stream#server_side_encryption-block

Reply:
For sure